src/dlgProfilePreferences.cpp

```diff
@@ -2072,44 +2072,55 @@ void dlgProfilePreferences::loadMap()
         return;
     }
 
-    QString fileName = QFileDialog::getOpenFileName(
-                           this,
-                           tr("Load Mudlet map"),
-                           mapSaveLoadDirectory(pHost),
-                           tr("Mudlet map (*.dat *.json);;Xml map data (*.xml);;Any file (*)",
-                              "Do not change extensions (in braces) as they are used programmatically"));
-    if (fileName.isEmpty()) {
-        return;
-    }
+    auto loadExtensions(QStringList()
+        <<   tr("Mudlet binary map (*.dat)", "Do not change extensions (in braces) as they are used programmatically")
+        <<   tr("Mudlet JSON map (*.json)", "Do not change extensions (in braces) as they are used programmatically")
+        <<   tr("Mudlet XML map (*.xml)", "Do not change extensions (in braces) as they are used programmatically")
+        <<   tr("Any map file (*.dat *.json *.xml)", "Do not change extensions (in braces) as they are used programmatically")
```

Comment:
I think this should be FIRST in the list - so that the user, by default, sees ALL the map files that Mudlet could load.

Note that using `loadMap()` with no arguments, from the Lua sub-system will load the *newest* readable file from the profiles map directory. However, strangely, I can not find *anything* about specify the sorting order in the dialogue, so we will just have to show them all the ones they could load and let them work out how to order them manually...

:bulb: Although (in a separate PR) it might be an idea if we could preselect the "newest" suitable file in the directory for them. :grinning:

Reply:
Fine point. Moved up.